memflow-win32/src/win32/keyboard.rs

```diff
@@ -337,6 +615,17 @@ impl<T> Win32Keyboard<T> {
                 .log_error("signature scanning requires std"),
         )
     }
+
+    #[cfg(not(feature = "regex"))]
+    fn find_global_slots_sig(
+        virt_mem: &mut impl MemoryView,
+        win32kbase_module_info: &ModuleInfo,
+    ) -> Result<umem> {
+        Err(
+            Error(ErrorOrigin::OsLayer, ErrorKind::UnsupportedOptionalFeature)
+                .log_error("signature scanning requires std"),
```

Comment:
Maybe an error message for end users like "cannot find keyboard signature because regex feature is disabled" would be more helpful